src/main.rs

```diff
@@ -214,30 +214,29 @@ impl PluginifyCommand {
             eprintln!("...package exists = {}", package.exists());
         }
 
-        let filename = package.file_name().ok_or_else(|| anyhow!("Can't get filename of {}", package.display()))?;
         let tar_path = PathBuf::from(format!("{}-{}-{}-{}.tar.gz", ps.name, ps.version, os, arch)).absolutize()?.to_path_buf();
         if self.verbose {
             eprintln!("About to create tar archive at {}", tar_path.display());
         }
 
         if self.verbose {
-            eprintln!("Appending {} with name {:?}", package.display(), filename);
+            eprintln!("Appending {} with name {:?}", package.display(), ps.name);
```

Comment:
This might need to be Windowsified I think?  (This wasn't an issue with `package.file_name` because a prior step Windowsified that already.)